consoles/localXvnc.pm

```diff
@@ -38,9 +38,7 @@ sub callxterm {
     die('Missing "icewm"')        unless which('icewm');
     die('Missing "xterm"')        unless which('xterm');
     eval { system("DISPLAY=$display $xterm_vt_cmd -title $window_name -e bash -c '$command' & echo \"xterm PID is \$!\""); };
-    if (my $E = $@) {
-        die "cant' start xterm on $display (err: $! retval: $?)";
-    }
+    die "cant' start xterm on $display (err: $! retval: $?)" if $@;
```

Comment:
I'm wondering when `$@` is actually set here (and why isn't it used in the error message then?)
Instead it should check `$?` I guess, or save the return value of `system`.

Reply:
I would like to preserve functionality for now in this PR but will try to followup.